modules/ROOT/pages/persistence-gateway.adoc

```diff
@@ -0,0 +1,154 @@
+= Persistence Gateway
+
+To store state across applications, Anypoint Runtime Fabric provides the Persistence Gateway. Persistence Gateway enables Mule applications deployed to a Mule runtime engine (Mule) to store and share data across replicas and application restarts.
+
+[NOTE]
+====
+Persistence Gateway is supported on Anypoint Runtime Fabric, version 1.9 or later and Mule runtime engine, version 4.2.1 or later.
+====
+
+== Use Persistence Gateway in a Mule Application
+
+After Persistence Gateway is configured in Anypoint Runtime Fabric, it is available for Mule applications deployed to Mule runtime engine, version 4.2.1 or later. Mule applications use the Object Store REST API to connect to the Persistence Gateway. This enables you to deploy to both Anypoint Runtime Fabric and CloudHub without having to modify your Mule application. See xref:deploy-to-runtime-fabric.adoc[Deploy a Mule Application to Runtime Fabric] for more information.
+
+When an application using persistent object storage is deleted, Runtime Fabric deletes all information stored in the Persistence Gateway.
+
+== Persistent Gateway Limits
+
+The following table lists the limits on the data stored by the Persistence Gateway:
+
+[%header,cols="2*a"]
+|===
+| Limit | Description
+| Maximum TTL. | The amount of time data is stored in the Persistence Gateway, value is set to 30 days.
+|===
+
+
+[Configure-Persistence-Gateway]
+== Configure Persistence Gateway
+
+To configure Persistence Gateway, you must create a Kubernetes custom resource that allows the cluster to connect to your persistence data store.
+
+=== Prerequisties
+
+Before enabling Persistence Gateway, ensure that you have created a database to serve as the data source for data stored by the Persistence Gateway. This database must be compatible with PostgresSQL 9 and above. 
+
+=== Create a Kubernetes Secret and Kubernetes Custom Resource
+
+. Create a Kubernetes secret:
+.. Copy the Kubernetes secret template from <<k8-secret-template>> to a file called `kubernetes-secret.yaml`.
+.. Create a base64-encoded connection string. 
++
+The standard URI for this string is `postgres://username:pass@host:port/databasename`. However, the requirements for your database may be different.
```

Comment:
@sleslie23 the `port` from connection string above is the one which would need to be allowed in the security group settings so it is reachable from object-store pod

Reply:
Added a new step:
Verify that the pod containing the persistence gateway service can connect to the database and port.

Reply:
Also added the following to install-prereqs.adoc and install-self-managed.adoc

==== Port Used by the Persistence Gateway

The Persistent Gateway requires a Postgres-compliant database to store persistent data across Mule application replicas. Ensure that your Kubernetes cluster has access to this database and port. See xref:persistence-gateway.adoc[Persistence Gateway].